code/modules/mob/living/simple_animal/hostile/netherworld.dm

```diff
@@ -0,0 +1,73 @@
+/mob/living/simple_animal/hostile/netherworld
+	name = "creature"
+	desc = "A sanity-destroying otherthing from the netherworld."
+	icon_state = "otherthing"
+	icon_living = "otherthing"
+	icon_dead = "otherthing-dead"
+	health = 80
+	maxHealth = 80
+	obj_damage = 100
+	melee_damage_lower = 25
+	melee_damage_upper = 50
+	attacktext = "slashes"
+	attack_sound = 'sound/weapons/bladeslice.ogg'
+	faction = list("creature")
+	speak_emote = list("screams")
+	gold_core_spawnable = 1
```

Comment:
TRUE

Reply:
i believe this is false, doesn't gold_core_spawnable = 2 mean it spawns on hostile reaction only?